### ISSUE TITLE
Support for @graph schema.org definitions

### ISSUE THREAD

**Issue by markusmo3** [obsidianmd/obsidian-clipper#127]
I'd like to be able to clip websites that only have a `@graph` schema.org definition.

Here is an example of a german pumpkin soup recipe i wanted to clip: https://www.eatbetter.de/rezepte/klassische-kuerbissuppe-nach-omas-rezept

I more or less tried everything there is to get some of the contents or even the template triggers to work. The suggested things from the obsidian clipper popup do not work and neither did my other tries:

- `{{schema:[0]:@graph[0].recipeIngredient}}` (this is the suggested one)
- `{{schema[0]:@graph[0].recipeIngredient}}`
- `{{schema[*]:@graph[0].recipeIngredient}}`
- `{{schema:@graph[0].recipeIngredient}}`

The schema.org validator seems to be able to parse it without a problem: https://validator.schema.org/#url=https%3A%2F%2Fwww.eatbetter.de%2Frezepte%2Fklassische-kuerbissuppe-nach-omas-rezept

It only has one schema script in the header:

```json
{
    "@context":"https://schema.org",
    "@graph":[
        {
            "@type":"Recipe",
            "name":"Klassische Kürbissuppe – nach Omas Rezept",
            "description":"Unser Rezept für klassische Kürbissuppe lässt die Herzen aller Kürbis-Fans höherschlagen. Hokkaido, Kartoffeln, Zwiebeln, Knoblauch und Sahne verbinden sich hier zu einer herrlich cremigen Suppe. Petersilie und Muskatnuss verleihen dem Klassiker den letzten Schliff. Ein Gericht mit Wohlfühl- und glücklich-mach-Garantie inklusive!",
            "keywords":"Rezepte,Suppen,Kürbissuppe,Kürbisrezepte,Vegetarische Rezepte,Hauptgerichte,Gemüserezepte,Einfache Rezepte,Gemüse Rezepte,Saison,Herbst,Gemüsesuppen,Cremesuppen,Vegetarische Suppen,Hokkaidokürbis,Kartoffel,Zwiebel,Knoblauchzehe,Butter,Gemüsebrühe,Petersilie,Sahne,Salz,Pfeffer,Muskatnuss,Vollkornbrot",
            "recipeCuisine":"deutsche Rezepte",
            "recipeYield":[
                "4",
                "4 Portionen"
            ],
            "dateModified":"2023-10-25T09:40:21+02:00",
            "datePublished":"2022-01-21T07:27:59+01:00",
            "prepTime":"PT0M",
            "cookTime":"PT35M",
            "totalTime":"PT35M",
            "recipeIngredient":[
                "1 Hokkaidokürbis (ca. 800 g)",
                "2 Kartoffel (ca. 200 g)",
                "2 Zwiebel",
                "1 Knoblauchzehe",
                "2 EL Butter",
                "1 L Gemüsebrühe (am besten selbst gekocht",
                "alternativ Bio-Instant-Brühe)",
                "0.5 Bd. Petersilie",
                "100 ml Sahne",
                "Salz",
                "Pfeffer",
                "frisch geriebene Muskatnuss",
                "8 Scheibe Vollkornbrot"
            ],
            "recipeInstructions":[
                "1. Kürbis waschen entstielen und halbieren. Mithilfe eines Löffels das Fruchtfleisch des Kürbis samt Kernen entfernen. Kartoffeln schälen. Kürbishälften und Kartoffeln in grobe Stücke schneiden. Zwiebeln abziehen und würfeln Knoblauch abziehen und fein hacken.",
                "2. Butter in einem großen Topf erhitzen. Zwiebeln darin 2-3 Minuten anbraten dann gehackten Knoblauch sowie Kürbis- und Kartoffelstücke zufügen kurz mitbraten und mit Gemüsebrühe ablöschen.",
                "3. Suppe aufkochen und bei mittlerer Hitze mit aufgelegtem Deckel ca. 20 Minuten köcheln lassen bis Kürbis und Kartoffeln weich ist. In der Zwischenzeit Petersilie abbrausen trocken schütteln und fein hacken.",
                "4. Suppe nach Ende der Garzeit mit einem Pürierstab fein pürieren mit Sahne verfeinern und je nach gewünschter Konsistenz noch ein wenig Wasser oder Gemüsebrühe zufügen. Fertige Suppe mit Salz Pfeffer sowie Muskatnuss abschmecken mit gehackter Petersilie garnieren und mit dem Vollkornbrot servieren."
            ],
            "recipeCategory":"Kürbissuppe",
            "author":{
                "@type":"Person",
                "name":"Simonetta Zieger – Ernährungswissenschaftlerin",
                "url":"https://www.eatbetter.de/autoren/simonetta-zieger"
            },
            "image":{
                "@type":"ImageObject",
                "representativeOfPage":"True",
                "url":"https://www.eatbetter.de/sites/eatbetter.de/files/styles/facebook/public/2021-08/13-kuerbissuppe-quer-4847.jpg?h=92704f59\u0026itok=ysvbihjT",
                "width":"1200",
                "height":"630"
            },
            "nutrition":{
                "@type":"NutritionInformation",
                "servingSize":"Portion",
                "calories":"428",
                "carbohydrateContent":"57.00",
                "proteinContent":"12.00",
                "fatContent":"14.00"
            },
            "aggregateRating":{
                "@type":"AggregateRating",
                "ratingValue":"4.90667",
                "ratingCount":"150",
                "bestRating":"5",
                "worstRating":"1"
            },
            "publisher":{
                "@type":"Organization",
                "name":"eatbetter",
                "url":"https://www.eatbetter.de",
                "logo":{
                    "@type":"ImageObject",
                    "url":"https://www.eatbetter.de/assets/images/logo-eatbetter-60.jpg",
                    "width":78,
                    "height":60
                },
                "address":{
                    "@type":"PostalAddress",
                    "addressCountry":"DE",
                    "addressLocality":"Offenburg",
                    "postalCode":"77652",
                    "streetAddress":"Hubert Burda Platz 1"
                }
            }
        }
    ]
}
```

**Comment by kepano** [comment]
This will be fixed in the next version.

**Comment by kepano** [comment]
Should be fixed in 0.9.5
